src/library/dao/trackdao.cpp

```diff
@@ -840,12 +860,12 @@ TrackPointer TrackDAO::addTracksAddFile(const TrackFile& trackFile, bool unremov
 bool TrackDAO::hideTracks(
         const QList<TrackId>& trackIds) const {
     QStringList idList;
-    for (const auto& trackId: trackIds) {
+    for (const auto& trackId : trackIds) {
         idList.append(trackId.toString());
     }
-    FwdSqlQuery query(m_database, QString(
-            "UPDATE library SET mixxx_deleted=1 WHERE id in (%1)").arg(
-                    idList.join(",")));
+    FwdSqlQuery query(m_database,
+            QString("UPDATE library SET mixxx_deleted=1 WHERE id in (%1)")
```

Comment:
SQtringLiteral please check the whole file. 

It looks  like you accidentally have reformatted it ...

Reply:
I rather prefer to undo all the automatic reformattings (hook broken?) instead of editing the legacy code. This PR should only contain changes that are directly related to the feature.

Reply:
Yes that's fine.